src/react-guides/Guides.tsx

```diff
@@ -199,6 +199,7 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
   private dragPositionElement() {
     const { className } = this.props as Required<GuidesProps>;
     return (
+      // eslint-disable-next-line max-len
       this.props.displayDragPos && <div className={`${prefix('wrapper-pos')} ${DISPLAY_DRAG} ${prefix(className)}`} ref={ref(this, 'displayElement')} style={this.props.dragGuideStyle} />
```

Comment:
Maybe better split this line something like this?
```
      this.props.displayDragPos && (
        <div
          className={`${prefix('wrapper-pos')} ${DISPLAY_DRAG} ${prefix(className)}`}
          ref={ref(this, 'displayElement')}
          style={this.props.dragGuideStyle}
        />
      )
```

Reply:
fixed